include/common/box.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_BOX_H
+#define LABWC_BOX_H
+
+#include <wlr/util/box.h>
+
+bool
+wlr_box_contains(struct wlr_box *box_super, struct wlr_box *box_sub);
```

Comment:
I prefer leaving `wlr_` out here on the basis that `wlr_` should be reserved for the `wlroots` API.

Aside: see 3rd paragraph of https://github.com/labwc/labwc/blob/master/CONTRIBUTING.md#naming-conventions

Reply:
Change wlr_ to lab_, for the new functions?'s, is t